ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -309,7 +313,8 @@ export const Homepage = fp.flow(withUserProfile(), withNavigation)(class extends
       });
       // TODO(RW-6493): Update rasCommonsLinked similar to what we are doing for eraCommons
 
-      const {workbenchAccessTasks} = queryParamsStore.getValue();
```

Comment:
I don't know why or when this stopped working.  Related to Angular changes maybe?